File: matiasmasso/Mat.NET/DTO/DTOProductSkuExtended.cs
using System;

namespace DTO
{
    public class DTOProductSkuExtended : DTOProductSku
    {
        public int UnitsInStock { get; set; }
        public int UnitsInClients { get; set; }
        public int UnitsInPot { get; set; }
        public int UnitsInProveidor { get; set; }
        public int UnitsInPrevisio { get; set; }
        public int Confirmed { get; set; }

        public DTOProductSkuExtended(Guid oGuid) : base(oGuid)
        {
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOProductSkuExtended.vb
Public Class DTOProductSkuExtended
    Inherits DTOProductSku

    Public Property UnitsInStock As Integer
    Public Property UnitsInClients As Integer
    Public Property UnitsInPot As Integer
    Public Property UnitsInProveidor As Integer
    Public Property UnitsInPrevisio As Integer
    Public Property Confirmed As Integer

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class
